adoc/Release-Notes.adoc

```diff
@@ -123,7 +123,7 @@ labels:
 
 * Deletion of {mariadb} instances created with Minibroker can fail with timeouts. If an error appears, wait one minute and retry. If the `cf delete-service` command fails but the instance pods are removed from {k8s}, the service instance data can safely be removed with a `cf purge-service-instance` command.
 
-* On {azure} it is recommended to run on instance types DS4_v2 or larger due to the introduction of the cflinuxfs3 stack.
+* On {azure} it is recommended to run on instance types `Standard_DS4_v2` or larger due to the introduction of the cflinuxfs3 stack. It's also recommended to use Premium SSD for the storage class.
```

Comment:
Maybe add (i.e., storage account type = `Premium_LRS`) at the end of the sentence. This should be covered in the documentation as well.